src/container/BooksForm.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const BooksForm = () => {
+  const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+  return (
+    <form>
+      <input type="text" />
+      <select>
+        {
+            categories.map((e) => <option key={Math.random()} value={e}>{e}</option>)
+          }
+      </select>
+      <button type="submit">Submit</button>
+    </form>
```

Comment:
- [ ] <b>OPTIONAL:</b> It would be nice to validate this form and make sure that it can't be submitted without filling all fields.